cmd/apply.go

```diff
@@ -50,6 +50,7 @@ func NewApplyCmd(globalCfg *config.GlobalImpl) *cobra.Command {
 	f.StringArrayVar(&applyOptions.Suppress, "suppress", nil, "suppress specified Kubernetes objects in the diff output. Can be provided multiple times. For example: --suppress KeycloakClient --suppress VaultSecret")
 	f.BoolVar(&applyOptions.SuppressSecrets, "suppress-secrets", false, "suppress secrets in the diff output. highly recommended to specify on CI/CD use-cases")
 	f.BoolVar(&applyOptions.ShowSecrets, "show-secrets", false, "do not redact secret values in the diff output. should be used for debug purpose only")
+	f.BoolVar(&applyImpl.ApplyOptions.NoHooks, "no-hooks", false, "do not diff changes made by hooks.")
```

Comment:
&applyImpl.ApplyOptions.NoHooks --> &applyOptions.NoHooks

Reply:
Maybe we can move the definition of `applyImpl := config.NewApplyImpl(globalCfg, applyOptions)` into the closure `RunE: func(cmd *cobra.Command, args []string) error {` so that we won't end up misusing it like this?

Reply:
good idea.